tensorhive/config.py

```diff
@@ -178,7 +197,39 @@ class PROTECTION_SERVICE:
     ENABLED = config.getboolean(section, 'enabled', fallback=True)
     UPDATE_INTERVAL = config.getfloat(section, 'update_interval', fallback=2.0)
     NOTIFY_ON_PTY = config.getboolean(section, 'notify_on_pty', fallback=True)
-
+    NOTIFY_VIA_EMAIL = config.getboolean(section, 'notify_on_pty', fallback=True)
```

Comment:
notify_on_pty should be changed to notify_via_email

Reply:
Should this option be disabled by default?